packages/client/src/render/screenRouter.js

```diff
@@ -3,6 +3,19 @@ import appStore from "../state/store"
 import { parseAppIdFromCookie } from "./getAppId"
 
 export const screenRouter = ({ screens, onScreenSelected, window }) => {
+  function sanitize(url) {
```

Comment:
NAB: can we re-use `screenTemplates/sanitizeUrl` here?

Reply:
it's slightly different - screenTempaltes one replaces spaces with "-" .. also in a different package, might be an issue